data/src/main/java/nl/tehdreamteam/se42/data/user/hibernate/UserHibernateRepository.java

```diff
@@ -24,22 +24,12 @@ public User get(String username) {
 
     @Override
     public void save(User user) {
-        Function<UserDAO, User> saveFunction = dao -> {
-            dao.create(user);
-            return null;
-        };
-
-        super.performTransaction(saveFunction);
+        super.create(user);
     }
 
     @Override
     public void remove(User user) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1185)